alonet/raft/raft.py

```diff
@@ -83,7 +86,7 @@ def build_update_block(self, update_cls=BasicUpdateBlock):
         """
         Build RAFT Update Block
         """
-        return update_cls(self.corr_levels, self.corr_radius, hidden_dim=self.hdim)
+        return update_cls(self.corr_levels, self.corr_radius, hidden_dim=self.hdim, out_planes=self.out_plane)
```

Comment:
Also here, if the child class do not implement out_plane, can we make sure theses values are != None and raise an exception otherwise